devtools-ci-index.yaml

```diff
@@ -2084,7 +2093,26 @@
             branches:
                 - origin/master
     <<: *rhche-functional-tests-template
-    
+
+- job-template:
+    name: '{ci_project}-{git_repo}-periodic-image-build'
+    wrappers:
+      - vault-secrets:
+          <<: *vault_defaults
+          secrets:
+            - *quay-eclipse-che-credentials
+    triggers:
+        - timed: '0 0 * * *'
+    publishers:
+        - email:
+            recipients: ibuziuk@redhat.com
```

Comment:
not sure what these recipients are but if it's for build failures notifications you may add more addresses here (e.g. Vitalii, Angel and myself)

Reply:
good catch